src/markdown.cpp

```diff
@@ -893,7 +894,22 @@ static int processLink(GrowBuf &out,const char *data,int,int size)
     else
     {
       out.addStr("<img src=\"");
-      out.addStr(link);
+      if (portable_isAbsolutePath(link) || isURL(link))
+      {
+        out.addStr(link);
+      }
+      else
+      {
+        // lets see if the file exists based on the path of the input file and the image file name (is a relative path / local filename)
+        QFileInfo fi(g_fileName);
+        QCString imgFile = g_fileName.left(g_fileName.length()-fi.fileName().length()) + link;
```

Comment:
The commit message says "requirement will be that the, doxygen related, path to the image file is in the IMAGE_PATH" but here I don't see image path being searched...

Reply:
Doesn't look like the `IMAGE_PATH` is involved. I have to do some extra tests.

Reply:
In case the `IMAGE_PATH` is not set the image is not copied to the HTML directory and thus not seen (unless the user copies it by "hand"). By setting the `IMAGE_PATH` the file is copied and is seen in the output. The copying is implicitly done outside the markdown code as the markdown image reference is converted to a `<img ...` tag.